docs/overview/project.rst

```diff
@@ -30,20 +33,26 @@ The flexibility provided allows to duplicate, delete, ignore and create content
 
 Every schema change is versioned, ensuring that the schema can be adapted to work with new requirements at any moment in time without breaking older analysis records.
 
-Depending on the preference and work environment of the researcher, analysis information can be created and edited through a `Submission Form`_ on the web interface or via the `REST API`_.
+Depending on the preference and work environment of the researcher, analysis information can be created and edited through a :ref:`project-submission-form` on the web interface or via the :ref:`project-rest-api`.
+
+.. _project-submission-form:
 
 Submission Form
 ~~~~~~~~~~~~~~~
 
-Submission forms are the graphical representation of the JSON schemas described in the last section. They allow manual editing of each field as well as inserting JSON conforming to the JSON schema of this part of the analysis form. Auto-complete and auto-fill will support editing fields for which information exists in one of the connected databases. See the section on `Connectivity`_ for more information.
+Submission forms are the graphical representation of the JSON schemas described in the last section. They allow manual editing of each field as well as inserting JSON conforming to the JSON schema of this part of the analysis form. Auto-complete and auto-fill will support editing fields for which information exists in one of the connected databases. See the section on :ref:`project-connectivity` for more information.
```

Comment:
I would propose "Information stored within the connected databases are used to support the analysis description by auto-completing and auto-filling, when ever that is possible."